examples/dreamcast/micropython/Makefile

```diff
@@ -0,0 +1,37 @@
+TARGET = micropython.elf
+
+CFLAGS += -std=c99
+CFLAGS += -I.
+CFLAGS += -Wall -Og
+
+SRCS += example.c
+OBJS += $(SRCS:.c=.o)
+OBJS += romdisk.o
+
+# The rm-elf step is to remove the target before building, to force the
+# re-creation of the rom disk.
+all: rm-elf $(TARGET)
+
+include $(KOS_BASE)/Makefile.rules
+
+clean: rm-elf
+	-rm -f $(OBJS)
+
+rm-elf:
+	-rm -f $(TARGET) romdisk.*
+
+$(TARGET): $(OBJS)
+	kos-cc -o $(TARGET) $(OBJS) -lmicropython
+
+romdisk.img:
```

Comment:
You can remove romdisk.img and romdisk.o from your makefile and stick 
`KOS_ROMDISK_DIR = romdisk`
 after OBJS += romdisk.o

Reply:
You can look at this file for an example:  https://github.com/KallistiOS/KallistiOS/blob/master/examples/dreamcast/vmu/vmu_game/Makefile